From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Mon, 18 Mar 2024 12:43:29 -0400
Subject: lower VTE dependency

vte 0.76 ought to have GTK 4.14 for accessibility improvements
But GNOME Terminal itself doesn't actually need vte 0.76
and doesn't use GTK4
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 5d80681..51189c2 100644
--- a/meson.build
+++ b/meson.build
@@ -45,7 +45,7 @@ clangxx_req_version       = '3.3'
 # Requirements
 
 vte_req                      = 'vte-2.91'
-vte_req_version              = '0.76.0'
+vte_req_version              = '0.74.0'
 
 glib_req_version             = '2.52.0'
 glib_min_req_version         = '2.52'
